servicetalk-opentelemetry-http/src/test/java/io/servicetalk/opentelemetry/http/OpenTelemetryHttpRequestFilterTest.java

```diff
@@ -99,14 +99,14 @@ void testInjectWithNoParent() throws Exception {
                 otelTesting.assertTraces()
                     .hasTracesSatisfyingExactly(ta ->
                         assertThat(ta.getSpan(0).getAttributes().get(SemanticAttributes.HTTP_URL))
-                            .startsWith("http://localhost:8080"));
+                            .isEqualTo("/"));
```

Comment:
I see this commit changes the values as well, in the commit message can you please explicitly list what has changed and to what? So people can track down changes later when they upgrade.